### ISSUE TITLE
Can we have on/off scripts to the events?

### ISSUE THREAD

**Issue by fascpt** [rasod/homebridge-dummy-garage#8]
I've been using "homebridge-cmdtriggerswitch" for a while to run scripts to both my gate and garage door (I have Python scripts which send specific commands via 433Mhz transmitter), but I've discovered that for Homekit purposes since they are seen as switches, anyone can control the gates using my Homepod "Hey Siri" commands, and that's a security issue. I've now learnt that if the accessory is set to Homekit as a proper Garage Gate, the Homepod needs the proper security validation before continuing.

Long story short, I'd love to use this plugin to trigger my scripts during the ON event. Since the plugin already supports auto turn off (and I don't care about states of the gates, since they can still be opened/closed with the physical remotes too), I'd like to know if you could potentially add script execution to both states (ON/OFF)?

Thanks!

**Comment by sebw** [comment]
Late response but create an automation that triggers homebridge-cmdtriggerswitch switches when the dummy garage door is actioned.
